docs/fedora-releases-guide.md

```diff
@@ -138,6 +138,27 @@ configuration key.
 
 ![Dashboard view for pull_from_upstream](img/fedora-releases-guide/pull-from-upstream-dashboard.png)
 
+#### Trying it out
+If you are interested in this functionality and want to try it out, you can trigger the job from a pull request.
+
+If there is a new release pending for your package ([release-monitoring.org](https://release-monitoring.org/) bugzilla has been opened but no rebase done in dist-git yet), do the following:
+
+- create a `rawhide`-based pull request with Packit configuration defining the [`pull_from_upstream` job](/docs/configuration/downstream/pull_from_upstream)
+- comment `/packit pull-from-upstream --with-pr-config` on the pull request
+- check the [dashboard](https://dashboard.packit.dev/jobs/pull-from-upstreams)
+- if everything went well, review the pull request(s) created by Packit
+- if you are happy with the results, merge your pull request and comment `/packit pull-from-upstream` to run the job for real (existing Packit-created pull requests will be updated)
```

Comment:
> comment `/packit pull-from-upstream` to run the job for real (existing Packit-created pull requests will be updated)

this should not be needed, or am I missing something?

Reply:
True, not really needed, I'll remove it.